generated/server/handlers.go

```diff
@@ -158,9 +172,10 @@ func CreateBookHandler(ctx context.Context, w http.ResponseWriter, r *http.Reque
 func NewCreateBookInput(r *http.Request) (*models.CreateBookInput, error) {
 	var input models.CreateBookInput
 
-	if err := json.NewDecoder(r.Body).Decode(&input.NewBook); err != nil {
-		return nil, err
-	}
+	var err error
+	_ = err
+
+	err = json.NewDecoder(r.Body).Decode(input.NewBook)
```

Comment:
nit: this code now looks much more auto-generated (I like the previous `if err := json.NewDecoder(...` better)